examples/predictors/lstm/xprize_predictor.py

```diff
@@ -91,82 +92,82 @@ def __init__(self, path_to_model_weights, data_url, cutoff_date_str):
 
         cutoff_date = pd.to_datetime(cutoff_date_str, format='%Y-%m-%d')
         self.df = self._prepare_dataframe(data_url, cutoff_date)
-        self.countries = self.df.CountryName.unique()
-        self.country_samples = self._create_country_samples(self.df, self.countries)
+        geos = self.df.GeoID.unique()
```

Comment:
Used a dedicated `GeoID` column instead of a "corrupted" `CountryName` one.
Makes the rest of this PR a little messy, but I'll point to the real changes